products/ol8/profiles/ospp.profile

```diff
@@ -161,7 +161,7 @@ selections:
     - auditd_freq
     - auditd_name_format
 
-    ### Module Blacklist
+    ### Module Disabled
```

Comment:
@ComplianceAsCode/oracle-maintainers Hi, are you okay with changes like these?

Reply:
No issue on our side 👍🏻